constants/statusCodes.go

```diff
@@ -0,0 +1,71 @@
+package constants
```

Comment:
these constant values are already in http package, can you reuse those
like this:
```
http.StatusOk
```

and StatusText is also defined, please take reference from here
https://go.dev/src/net/http/status.go

Reply:
something like this should be fine ig

```go
const HTTPStatusCodes := map[int]string{
	http.StatusOK:                   "OK",
	http.StatusCreated:              "Created",
	http.StatusAccepted:             "Accepted",
	http.StatusNonAuthoritativeInfo: "Non-Authoritative Information",
	http.StatusNoContent:            "No Content",
	http.StatusBadRequest:           "Bad Request",
	http.StatusUnauthorized:         "Unauthorized",
	http.StatusForbidden:            "Forbidden",
	http.StatusNotFound:             "Not Found",
	http.StatusMethodNotAllowed:     "Method Not Allowed",
	http.StatusInternalServerError:  "Internal Server Error",
}

fmt.Println(HTTPStatusCodes[http.StatusOK])          // Output: "OK"
```

Reply:
@loyaltytrooper you can directly use the above map

Reply:
It is in http package

```golang
http.StatusText(http.StatusOK) // Output: "OK"
```

Reply:
check line 80
https://go.dev/src/net/http/status.go

Reply:
Then
```go
HTTPStatusCodes := map[int]string{
		http.StatusOK:                   http.StatusText(http.StatusOK),
		http.StatusCreated:              http.StatusText(http.StatusCreated),
		http.StatusAccepted:             http.StatusText(http.StatusAccepted),
		http.StatusNonAuthoritativeInfo: http.StatusText(http.StatusNonAuthoritativeInfo),
		http.StatusNoContent:            http.StatusText(http.StatusNoContent),
		http.StatusBadRequest:           http.StatusText(http.StatusBadRequest),
		http.StatusUnauthorized:         http.StatusText(http.StatusUnauthorized),
		http.StatusForbidden:            http.StatusText(http.StatusForbidden),
		http.StatusNotFound:             http.StatusText(http.StatusNotFound),
		http.StatusMethodNotAllowed:     http.StatusText(http.StatusMethodNotAllowed),
		http.StatusConflict:             http.StatusText(http.StatusConflict),
		http.StatusInternalServerError:  http.StatusText(http.StatusInternalServerError),
	}
```

Reply:
Cool